lib/models/home_card_model.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:erobot/constant/theme_constant.dart';
+import 'package:flutter/material.dart';
+
+class HomeCardModel {
+  String title;
+  String subTitle;
+  IconData? icon;
+  Gradient? background;
+  bool isSmall;
+  Function()? onTap;
+
+  HomeCardModel({
+    this.title = '',
+    this.subTitle = '',
+    this.icon,
+    this.background,
+    this.isSmall = true,
+    this.onTap,
+  });
+
+  static getCards() => [
+        HomeCardModel(
+          title: 'Sender',
+          subTitle: 'Send string to arduino robot via Bluetooth',
+          icon: Icons.keyboard,
+          background: ThemeConstant.greenGradient,
+          isSmall: false,
```

Comment:
this should be `isBig`, since it will be a bigger card

Reply:
I think the default is small so I set it isSmall   😂